### DISCUSSION TITLE
Can i use URL in reponse for "send request" block?

### DISCUSSION THREAD

**Discussion by tlustopro** [postmanlabs/postman-flows#283] [Q&A] [answered]
Hi all,

i would like to use global variable as request URL. For example i call first request, which give me URL of image in JSON response. This URL i want to use in next block "send request" (or any other block which can send request).

Is it possible?

Thanks!

**Comment by samay-v** [comment, marked as answer]
Hey @tlustopro 
sorry for the late reply. I created a flow to explain how you can achieve this, check it out here https://www.postman.com/postman/workspace/postman-answers/flow/626b96e2c19f90442d48f1e4
